CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Changed
+- updated sensu-plugin gem to `~> 2.0`
```

Comment:
This is technically a breaking change as this pulls in the newer version of `sensu-plugin` which does have some compatibility issues with gems that rely on the `json` gem that is `< 2.0`. It needs to be moved down a little bit and use the `### Breaking Changes` header. Also it would be great if you could include a link to the relevant changelog in sensu plugins. For example:
```
### Breaking Changes
- bumped dependency of `sensu-plugin` to 2.x you can read about it [here](https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#v200---2017-03-29) (@multani)

### Fixed
- fix runtime warnings (@multani)
```

Reply:
> It needs to be moved down a little bit [...]

I'm not sure what you exactly mean?

Reply:
Oh my bad I must have got pulled away and when I came back did not stitch my thoughts together right.

So you have:
```
### Changed
￼- updated sensu-plugin gem to `~> 2.0`
- fix runtime warnings
```

What you should have based on the type of change and it's impact:
```
### Breaking Changes
- bumped dependency of `sensu-plugin` to 2.x you can read about it [here](https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#v200---2017-03-29) (@multani)

### Fixed
- fix runtime warnings (@multani)
```

Reply:
OK, that's what I thought but I wasn't sure :D

I pushed a change in bb1bbf9 which should address this.

Reply:
perfect, also I added to your changelog entry that it was runtime warnings for `timeout`.